lib/rubocop/cop/util.rb

```diff
@@ -148,6 +148,10 @@ def move_pos(src, pos, step, condition, regexp)
         pos
       end
 
+      def looking_at(range, regexp)
+        range.source_buffer.source.match(/\G#{regexp}/, range.begin_pos)
```

Comment:
Hope this added explanation works: https://github.com/ruby/ruby/commit/066b825400349c559aa3c1ca7769516c967c41b9